projects/RabbitMQ.Client/client/framing/TxSelect.cs

```diff
@@ -34,22 +34,16 @@
 
 namespace RabbitMQ.Client.Framing.Impl
 {
-    internal sealed class TxSelect : Client.Impl.MethodBase
+    internal readonly struct TxSelect : IOutgoingAmqpMethod
```

Comment:
For empty structs like the `Tx*` structs, we might as well create a static singleton since there is no reason to even new them up?

Reply:
Possible, but the benefits are very limited (new for this type of struct is "basically a noop" as it has a size of 0, the change in useage makes it more confusing than benefiting I think.